components/brave_rewards/resources/tip/components/publisher_banner.tsx

```diff
@@ -113,10 +113,22 @@ function getSocialIcon (type: string) {
   }
 }
 
+function isValidSocialLink (url: string) {
+  try {
+    // The URL constructor will throw when provided with any
+    // string that is not an absolute URL. If the URL constuctor
+    // does not throw, consider it a valid social link URL.
+    const _ = new URL(url)
```

Comment:
Do we even need to store the return value at all since we only care about the side effect?

Reply:
No - I was trying to make tslint happy, but just ended up using a tslint ignore comment.